japanese/beanus/beanus.md.txt

```diff
@@ -0,0 +1,4 @@
+﻿川の音を頼りに、森の中にぐんぐん分け入ってと、綺麗な泉を見つけたあなた。
```

Comment:
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.